homeassistant/components/advantage_air/entity.py

```diff
@@ -17,14 +20,29 @@ def __init__(self, instance: dict[str, Any]) -> None:
         super().__init__(instance["coordinator"])
         self._attr_unique_id: str = self.coordinator.data["system"]["rid"]
 
+    def update_handle_factory(self, func, *keys):
+        """Return the provided API function wrapped.
+
+        Adds an error handler and coordinator refresh, and presets keys.
+        """
+
+        async def update_handle(*values):
+            try:
+                if await func(*keys, *values):
+                    await self.coordinator.async_refresh()
```

Comment:
```suggestion
                    await self.coordinator.async_request_refresh()
```

So it debounce?

Reply:
The upstream library function returns **true** only when an API request was actually made, and there is value in refreshing. Any subsequent calls while there is still a pending API request return false because they are queued.

TLDR the debounce is handled by the library since it merges multiple request into one API call.

Reply:
https://github.com/Bre77/advantage_air/blob/master/advantage_air/__init__.py#L87-L89